.travis.yml

```diff
@@ -3,9 +3,5 @@ language: ruby
 rvm:
   - 2.1.4
```

Comment:
We've `.ruby-version` file https://docs.travis-ci.com/user/languages/ruby/#Using-.ruby-version